### PR TITLE
feat:(TON-2667) part 2 add tonhub api

### PR DESCRIPTION
**Pull request by sjDu**
## Summary

- add tonhub api to amend the missing functionality 

* This PR contains  https://github.com/frigatebird-studio/TonFura.js/pull/7
merge that one first to see the changes only in the PR

## Ticket(s)
https://tonfura.atlassian.net/browse/TON-2667

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor (no functionality changes)

## Checklist

- [ ] Remove unnecessary console or comment
- [ ] Add unit tests

### CODE REVIEW

File: packages/adapter/src/tonclient4/converters/convertGetConfig.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { GetConfigParamResponse, Config } from '../types';
+
+export function convertGetConfig(data: GetConfigParamResponse): Config {
+  return {
+    config: {
+      cell: data.result.config.bytes,
+      address: "", // todo we don't have such data
+      globalBalance: {
+        coins: "" // todo we don't have such data
```
Comment (WeiYiChiuAtTonfura) [inline packages/adapter/src/tonclient4/converters/convertGetConfig.ts]: 是不是代表要用  ton hub api?

File: packages/adapter/src/tonclient4/converters/convertRunMethod.ts [hunk 2]
```diff
@@ -7,11 +7,16 @@ export function convertRunMethod(data: RunGetMethodResponse, accountData: Accoun
...
     reader, // todo we don't have such data
     block: accountData.block,
-    shardBlock: undefined, // todo we don't have such data
+    shardBlock: {
+      workchain: -2, // todo we don't have such data
+      seqno: 0, // todo we don't have such data
+      shard: '0', // todo we don't have such data
+      rootHash: '', // todo we don't have such data
+      fileHash: '', // todo we don't have such data
+    }, // todo we don't have such data
```
Comment (WeiYiChiuAtTonfura) [inline packages/adapter/src/tonclient4/converters/convertRunMethod.ts]: 是不是代表要用  ton hub api?

File: packages/adapter/src/tonclient4/converters/convertSend.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Send, SendMessageResponse } from '../types';
+
+export function convertSendMessage(data: SendMessageResponse): Send {
+  return {
+    status: data.result? 1 : 0, // to check the status value
```
Comment (WeiYiChiuAtTonfura) [inline packages/adapter/src/tonclient4/converters/convertSend.ts]: 1, 0 的意義是什麼 🤔

File: packages/adapter/src/tonclient4/converters/convertGetConfig.ts (hunk 1 above)
Comment (sjDu) [reply packages/adapter/src/tonclient4/converters/convertGetConfig.ts]: 對

File: packages/adapter/src/tonclient4/converters/convertRunMethod.ts (hunk 2 above)
Comment (sjDu) [reply packages/adapter/src/tonclient4/converters/convertRunMethod.ts]: 對

File: packages/adapter/src/tonclient4/converters/convertSend.ts (hunk 3 above)
Comment (sjDu) [reply packages/adapter/src/tonclient4/converters/convertSend.ts]: 1 success
0 fail

**Review by WeiYiChiuAtTonfura — APPROVED** [review]

**Comment by sjDu** [comment]
discussion on slack:
https://frigatebird-studio.slack.com/archives/C041128BU7M/p1727947785891709?thread_ts=1727773931.490679&cid=C041128BU7M

**Pull request merged**